crates/precompile/Cargo.toml

```diff
@@ -64,9 +65,12 @@ c-kzg = { workspace = true, optional = true, features = [
 # Optionally use `kzg-rs` for a pure Rust implementation of KZG point evaluation.
 kzg-rs = { workspace = true, optional = true }
 
-# BLS12-381 precompiles
+# Use the BLS12-381 implementation of blst for EIP2537
 blst = { workspace = true, optional = true }
```

Comment:
Making blst optional but default because it doesn't compile with the riscv backend